src/components/Text/Text.mdx

```diff
@@ -39,3 +42,11 @@ accessible and will be read by screen readers.
 Resources:
 
 - [Chakra UI Text](https://chakra-ui.com/docs/components/typography/text)
+
+## Updated Typographic Styles
+
+New typographic styles were introdiuced in June 2023. The full array of size options include:
+
+`subtitle1`, `subtitle2`, `overline1`, `overline2`, `default`, `caption`, `tag`, `mini`
```

Comment:
This should also be split up to say some are deprecated and not recommended to use.